Dockerfile-client

```diff
@@ -0,0 +1,53 @@
+FROM python:3.8-alpine3.12
+
+RUN apk --update --no-cache add \
+    build-base \
+    linux-headers \
+    curl \
+    make \
+    perl \
+    gcc \
+    tar \
+    libffi-dev \
+    python3-dev \
+    py3-virtualenv \
+    bash
+
+# Copy necessary files in the docker container
+COPY ./client /home/zeth
+
+ENV OPENSSL_VERSION="1.1.1g"
+ENV CWD="/home/zeth"
```

Comment:
Should this be something like `BUILD_ROOT` or similar?  I had to pause and understand since `CWD` is so strongly tied to current working directory.

Reply:
See: https://github.com/clearmatics/zeth/pull/245/commits/bdf9b884f5d7c190635f18b8c63018b290e57225